src/components/MoneyRequestHeader.js

```diff
@@ -85,7 +85,9 @@ const MoneyRequestHeader = (props) => {
         : ReportUtils.getAvatar(lodashGet(props.personalDetails, [moneyRequestReport.managerEmail, 'avatar']), moneyRequestReport.managerEmail);
     const policy = props.policies[`${ONYXKEYS.COLLECTION.POLICY}${props.report.policyID}`];
     const shouldShowSettlementButton =
-        !isSettled && (Policy.isAdminOfFreePolicy([policy]) || (ReportUtils.isMoneyRequestReport(props.report) && lodashGet(props.session, 'email', null) === props.report.managerEmail));
+        !isSettled &&
+        !props.isSingleTransactionView &&
+        (Policy.isAdminOfFreePolicy([policy]) || (ReportUtils.isMoneyRequestReport(props.report) && lodashGet(props.session, 'email', null) === props.report.managerEmail));
```

Comment:
That's a whole lotta logic. Maybe consider a helper or separate condition within the component for the final line

Reply:
split it up a bit!